### DISCUSSION TITLE
Homework_01 Part 2

### DISCUSSION THREAD

**Discussion by arodema** [cooperrc/engineering-dynamics#8] [General]
Hello! I'm a little confused on what exactly is expected from me on Homwork_01 Part 2. Would anybody be able to clarify please?

**Comment by leena-alam** [comment]
I think we need to give an equation for x(t) of the three scenarios and then graph those lines! He shows us how to do it in the online lecture from today (1/29)

**Reply by scoutfavreau** [reply]
do we have to graph it on jupyter hub like he does or can we graph it on paper

**Reply by cooperrc** [reply]
You should use Python, Matlab, Excel, or any other plotting software. 

People used to plot these by hand, but it takes a lot of practice to get it to look right. Its much easier to plot with software now
